hieradata/role/webserver.yaml

```diff
@@ -13,6 +15,14 @@ psick::profiles::windows_classes:
 #  firewall: windows_firewall
 #  users: psick::users
 
+psick::packages::resource_default_arguments:
+  provider: chocolatey
+psick::packages::packages_list:
+  - "chocolatey"
```

Comment:
is this needed? shoudnl't be chocolatey already installed via the chocolatey class?

Reply:
@alvagante A question on chocolatey vs `psick::windows::features` - chocolately seems to handle many bases well. Is there a reason for us to use both?

Reply:
well, windows features manages only  windows  components, chocolately manages third party applications. I don't know if and how well chocolately manages windows features, if it does it well we might use it for everything

Reply:
So far, my experience with it has been super-positive, but obviously may not do all you need.